app/models/user.rb

```diff
@@ -1,5 +1,10 @@
 # app/models/user.rb
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] [OPTIONAL] You have done a great job @NoelLincoln . Confirmable works jut fine 👍🏽

Reply:
Thank you